plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -191,7 +192,13 @@ def sweep_over_batches(
             )
             self.job_idx += len(rets)
             for idx in range(len(batch)):
-                val = rets[idx].return_value
+                val: Any = rets[idx].return_value
+                assert isinstance(val, (int, float, tuple))
```

Comment:
Add a comment to this assert. I think it will trip some people.
something like:
```python
assert isinstance(val, (int, float, tuple)), f"Return value type {type(val).__name__} is not supported by Ax"
```